openprescribing/outliers/views.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+from django.conf import settings
+from django.shortcuts import render
+from frontend.models import Chemical, Section
+from frontend.views.views import _entity_type_human, _get_entity
+
+
+def outliers_for_one_entity(request, entity_type, entity_code):
+    entity = _get_entity(entity_type, entity_code)
+    base_analyse_url = f"/analyse/#org={entity_type}&orgIds={entity_code}"
+    bnf_code_to_name = dict(
+        list(Chemical.objects.values_list("bnf_code", "chem_name"))
+        + list(Section.objects.values_list("bnf_id", "name"))
+    )
+    df = pd.read_feather(
+        settings.OUTLIERS_DATA_DIR / entity_type / f"{entity_code}.feather"
+    )
```

Comment:
Given that we're expecting these files to sometimes not exist it would be nice to handle that gracefully here, mainly to avoid the Sentry spam we'll get otherwise. 

By "gracefully" I just mean a 404 – possibly with a message along the lines of "we're reimporting data please check back in a few minutes", if it's easy to include.